modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/node/metric/NodeMetricDisableCommand.java

```diff
@@ -48,6 +49,7 @@ public Integer call() {
                 .output(spec.commandLine().getOut())
                 .errOutput(spec.commandLine().getErr())
                 .verbose(verbose)
+                .exceptionHandler(new ClusterNotInitializedExceptionHandler("Cannot disable metrics", "cluster init"))
```

Comment:
```suggestion
                .exceptionHandler(new ClusterNotInitializedExceptionHandler("Cannot disable metrics", "ignite cluster init"))
```

Reply:
In case this is the REPL command the `ignite` prefix does not make ane sence.

Reply:
> In case this is the REPL command the `ignite` prefix does not make ane sence.

But it's not a REPL command, there's a separate `NodeMetricDisableReplCommand`

Reply:
I would create a ticket and simply extract a factory method which will encapsulate the command string.